simvue_fds/connector.py

```diff
@@ -451,6 +602,13 @@ def check_for_errors(status_code, std_out, std_err):
             completion_callback=check_for_errors,
         )
 
+        if self.slice_parse_quantity:
+            slice_parser = threading.Thread(target=self._slice_parser)
```

Comment:
Consider naming the thread (forgot the argument name but there is one) and just set `daemon=True` in the initialiser, you do not need the line below at all